server/main.go

```diff
@@ -25,10 +26,13 @@ type server struct {
 }
 
 func main() {
+
 	var tcpAddr string
 	var unixPath string
+	var nlEventTracker bool
 	flag.StringVar(&tcpAddr, "tcp", "", "Run as a TCP server and listen on target address")
 	flag.StringVar(&unixPath, "unix", "", "Run as a UNIX server and listen on target path")
+	flag.BoolVar(&nlEventTracker, "nl", false, "Run as a Netlink Event Tracker")
```

Comment:
I would like to propose the flag names and usage descriptions
```go
flag.BoolVar(&nlEventTracker, "track-netlink", false, "Run as a netlink event tracker for garbage collection")

// OR

flag.BoolVar(&nlEventTracker, "netlink-gc", false, "Run as a netlink event tracker for garbage collection")
```
the command can be used like

`./network-controller -unix=/tmp/grpc.sock -track-netlink=true`

OR 

`./network-controller -unix=/tmp/grpc.sock -netlink-gc=true`